Feat/i18n

durancristhian opened the pull request.

durancristhian committed:
Add basic next translate implementation

durancristhian committed:
Basic Layout

durancristhian committed:
Improve config

mateosilguero committed:
Merge branch 'feat/i18n' of github.com:durancristhian/coronabingo into feat/i18n

durancristhian committed:
Add Select on header + first translation

mateosilguero committed:
Merge branch 'feat/i18n' of github.com:durancristhian/coronabingo into feat/i18n

mateosilguero committed:
jugar page, footer

durancristhian committed:
More translations

durancristhian committed:
Merge branch 'feat/i18n' of github.com:durancristhian/coronabingo into feat/i18n

durancristhian committed:
fix

durancristhian commented on components/Boards.tsx:
```diff
@@ -57,7 +60,9 @@ export default function Boards({ player, setPlayerProps }: IProps) {
         >
-          <p className="font-semibold uppercase">Cartón Nº {board.id}</p>
+          <p className="font-semibold uppercase">
+            {t('common:board')} Nº {board.id}
```
Esto conviene meterlo todo en el locale y que sea parametrizable, más que nada porque en algunos idiomas nos va a cagar la vida creo

durancristhian commented on components/BackgroundCells.tsx:
```diff
@@ -1,5 +1,7 @@
 import classnames from 'classnames'
 import { Fragment, useContext } from 'react'
+// @ts-ignore
```
Tenemos que crear la definición básica de este módulo en un archivo `*.d.ts` porque el comentario ignorando el import que tenemos que meter en cada fucking file nos va a dejar (más) pelados

mateosilguero replied:
Mas pelado no se puede jajaj, pero done

durancristhian commented on components/Players.tsx:
```diff
@@ -2,7 +2,7 @@ import classnames from 'classnames'
 import { FiPlus, FiTrash2 } from 'react-icons/fi'
 import Select from '~/components/Select'
-import { Field } from '~/pages/sala/[name]/admin'
+import { Field } from '~/pages_/sala/[name]/admin'
```
Mover este tipo a otro lado, evitar los imports entre páginas para estas cosas

durancristhian commented on locales/en/jugar.json:
```diff
@@ -0,0 +1,15 @@
+  "next-number": "Next number",
+  "no-turningglob-description": "Mark in the list below the numbers that come out in your turning globe.",
+  "admin-title": "You are viewing this panel because you are the one who runs the room",
+  "celebrate": "Let's celebrate !",
```
```suggestion
  "celebrate": "Let's celebrate!",
```

durancristhian committed:
Update locales/en/jugar.json

mateosilguero committed:
sala page, fix on header selector redirect

durancristhian committed:
Finished home translations

durancristhian committed:
Finished admin translations

mateosilguero committed:
resolve comments. implement ts interfaces

mateosilguero committed:
merge

mateosilguero committed:
remove ts-ignore

durancristhian committed:
Update translations

durancristhian committed:
Merge branch 'feat/i18n' of github.com:durancristhian/coronabingo into feat/i18n

durancristhian committed:
Finish translations

durancristhian committed:
Fix issue with build

durancristhian committed:
Improve select

durancristhian committed:
Fix select

durancristhian committed:
1.5.0

mateosilguero committed:
remove asPath, so the site will refresh on lang change

mateosilguero committed:
Merge branch 'feat/i18n' of github.com:durancristhian/coronabingo into feat/i18n

durancristhian committed:
Add texts

durancristhian committed:
Add language to the sounds

durancristhian committed:
Improve config

durancristhian committed:
Add trranslation

swaczuk committed:
Update SEO tags home

Modifiqué el title y description de redes y html. También agregué url canonical y sus alternativas de idiomas.

durancristhian committed:
Add trranslations for dreams

durancristhian committed:
fix

durancristhian committed:
Merge pull request #29 from durancristhian/langtags-home

Update SEO tags home

mateosilguero committed:
'english' sounds

durancristhian committed:
metatags

durancristhian committed:
Merge pull request #30 from durancristhian/english-sounds

'english' sounds

mateosilguero committed:
english meta description

durancristhian committed:
ya no se que mas

durancristhian committed:
cardi b

durancristhian committed:
cardi b 2

mateosilguero committed:
refresh home

durancristhian committed:
title
